src/main/cpp/Robot.cpp

```diff
@@ -60,7 +60,7 @@ void Robot::RobotInit()
 
     m_datalogger = DragonDataLoggerMgr::GetInstance();
 
-    auto path = AutonUtils::GetPathFromTrajectory("BlueLeftInside_I"); // load choreo library so we don't get loop overruns during autonperiodic
+    auto path = AutonUtils::GetTrajectoryFromPathFile("BlueLeftInside_I"); // load choreo library so we don't get loop overruns during autonperiodic
```

Comment:
We may want to check if we still need this.  It might have been path planner that was the issue.

Reply:
Yeah I agree, I hesitated on this line. I thought I would just replace it for now since it is in init and it probably doesn't hurt anything. Any ideas on the way to verify it? I can add timers, but I feel like there is a better way to do it? If you don't have another idea, I can just run it in the sim with and without this line and time each time we make this AutonUtils call

Reply:
I agree with your assessment.   We'd have to run and see if the first auton primitive has an excessive time